homeassistant/components/enphase_envoy/sensor.py

```diff
@@ -259,6 +260,36 @@ class EnvoyEnchargePowerSensorEntityDescription(
 )
 
 
+@dataclass
+class EnvoyEnpowerRequiredKeysMixin:
+    """Mixin for required keys."""
+
+    value_fn: Callable[[EnvoyEnpower], datetime.datetime | int | float]
+
+
+@dataclass
+class EnvoyEnpowerSensorEntityDescription(
+    SensorEntityDescription, EnvoyEnpowerRequiredKeysMixin
+):
+    """Describes an Envoy Encharge sensor entity."""
+
+
+ENPOWER_SENSORS = (
+    EnvoyEnpowerSensorEntityDescription(
+        key="temperature",
+        native_unit_of_measurement=UnitOfTemperature.FAHRENHEIT,
```

Comment:
So weird that some endpoints use F and some use C

Reply:
I become less surprised the more I explore these API endpoints 🙃